experimental/vpx_express/custom_script.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+import logging
+import os
+import urllib2
+import time
+import json
+import socket
+import struct
+import base64
+import requests
+import sys
+
+
+# Globals
+AZURE_CUST_DATA_FILE = "/nsconfig/.AZURE/customdata"
+
+# Set logging
+logging.basicConfig()
```

Comment:
Is all log redirected to console? Can we append it to single separate file?

Reply:
logs are being finally written to /var/log/azure/../extension.log